meshapp/src/profile/profile-textfield.tsx

```diff
@@ -22,7 +22,7 @@ const ProfileTextField = (props: {
   placeholder: string;
   text: string;
   charLimit: number;
-  handleSave: any;
+  handleSave: (text: string)=> void;
```

Comment:
Just out of curiosity, why was this changed?

Reply:
The method still works with type any, but TypeScript says it's best practice to clarify a specific type unless the data type is unknown.